src/utils.js

```diff
@@ -32,7 +32,7 @@ let validateResolver = (resolver) => {
 let rateStrRegex = /\d+\/(ns|us|µs|ms|s|m|h)/;
 
 let isValidRateStr = (rate) => {
-    if (typeof rate !== 'string') {
+    if (typeof rate !== 'string' && !(rate instanceof String)) {
```

Comment:
I don't think this check is totally necessary since it is generally considered bad practice to use the String class.

Reply:
I referenced to [this](https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string-in-javascript#:~:text=I%20strenuously%20disagree,at%204%3A11) StackOverflow discussion. Which is I think it is better to handle all possibility. But yeah you right that using String class is actually a _very unusual_ coding practice.

Reply:
Ahh got it!  Works for me and yeah it makes sense.